src/components/Button/Button.tsx

```diff
@@ -92,10 +92,20 @@ class Button extends UIComponent<any, any> {
     accessibility: ButtonBehavior as Accessibility,
   }
 
-  public renderComponent({ ElementType, classes, accessibility, rest }): React.ReactNode {
+  public renderComponent({
+    ElementType,
+    classes,
+    accessibility,
+    variables,
+    rest,
+  }): React.ReactNode {
     const { children, content, disabled, iconPosition } = this.props
     const hasChildren = childrenExist(children)
 
+    if (hasChildren) {
+      return children
+    }
```

Comment:
If I now do:

```jsx
<Button>Click me</Button>
```

Won't this result in rendering the text "Click me" only?  We'd still want to wrap the text in an  `ElementType` and apply our classes/accessibility/rest props.